com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.Skin.cs

```diff
@@ -44,6 +44,7 @@ sealed class Styles
             public readonly string cookieNonPOT = "HDRP does not support non power of two cookie textures.";
             public readonly string cookieTooSmall = "Min texture size for cookies is 2x2 pixels.";
             public readonly string cookieBaking = "Light Baking for cookies disabled on the Project Settings.";
+            public readonly GUIContent includeLightForRayTracing = new GUIContent("Include For RayTracing", "When checked, the light will affect the scene for a camera that has the Ray-Tracing frame setting enabled.");
```

Comment:
>"Include For RayTracing"
@JordanL8  We are trying to find a label to say that a light is enabled when performing a rendering with a Camera that have a Framesettings "Raytracing" enable.  What do you think is the best wording for this? is Influce for Raytracing good? thanks

Reply:
this was discussed with lewis before the PR

Reply:
Yeah I thought Include For Raytracing made sense. For the tooltip, how about:

When enabled, the light affects the scene for cameras that have the Ray-Tracing frame setting enabled.